src/main/java/by/bookify/image/controller/ImageController.java

```diff
@@ -14,7 +16,7 @@
 @RequiredArgsConstructor
 public class ImageController {
 
-    private final ImageService imageService;
+    private final GridFsMongoImageService imageService;
```

Comment:
You have choosen the version explicitly. It's better to use contract in other words interfaces. Use Qualifier annotation. You can read about it here: https://www.baeldung.com/spring-qualifier-annotation